Casks/anka-virtualization@2.4.1.130.602476-nestedBeta5.rb

```diff
@@ -0,0 +1,45 @@
+cask "anka-virtualization@2.4.1.130.602476-nestedBeta5" do
+  version "2.4.1.130.602476-nestedBeta5"
+  sha256 "94b904c45223a929818f59c80793ab20e6156be8bd6a25ccf4f81f78b78c4f30"
+
+  url "https://downloads.veertu.com/anka/Anka-#{version}.pkg",
+      verified: "downloads.veertu.com/anka/"
+  appcast "https://ankadocs.veertu.com/docs/release-notes/"
+  name "Anka Virtualization"
+  desc "CLI tool for managing and creating virtual machines"
+  homepage "https://veertu.com/"
+
+  depends_on macos: ">= :yosemite"
+
+  pkg "Anka-#{version}.pkg"
+
+  uninstall launchctl: [
+    "com.veertu.nlimit",
+    "com.veertu.vlaunch",
+    "com.veertu.anka.lupd",
+    "com.veertu.anka.ankakbd",
+    "com.veertu.anka.ankanetd",
+  ],
+            script:    {
```

Comment:
indentation is strange, but I guess it doesn't really matter

Reply:
yeah it is. I copied this from another file

I will possibly make a new PR to fix them all